rest_framework_extensions/mixins.py

```diff
@@ -1,8 +1,14 @@
 # -*- coding: utf-8 -*-
+# Try to import six from Django, fallback to included `six`.
+try:
+    from django.utils import six
+except ImportError:
```

Comment:
I just did the import the same way `rest_framework_extensions/compat.py` does it.

Django didn't ship with six until version 1.5.
https://docs.djangoproject.com/en/1.8/topics/python3/

I'm not sure when Django Rest Framework stopped shipping (or ever did ship with) it's own version of six.
I couldn't find a commit where `rest_framework/__init__.py` has a reference to six: https://github.com/tomchristie/django-rest-framework/commits/master/rest_framework/__init__.py

It's probably ok rely on Django's version of six.
I'm ok with whatever the drf-extensions package owner decides and can make the appropriate changes.